### PR TITLE
Extend (slightly) entry index interface [ECR-1832]

### PR DESCRIPTION
**Pull request by DarkEld3r**

### CODE REVIEW

**Comment by aleksuss** [comment]
What about to add some unittests ?

File: exonum/src/storage/entry.rs [hunk 1]
```diff
@@ -172,4 +172,56 @@ where
...
+    /// index.set(10);
+    /// assert_eq!(Some(10), index.get());
+    ///
+    /// let value = index.swap(20);
+    /// assert_eq!(Some(10), value);
+    /// assert_eq!(Some(20), index.get());
+    /// ```
+    pub fn swap(&mut self, value: V) -> Option<V> {
+        let previous = self.get();
+        self.set(val);
```
Comment (aleksuss) [inline exonum/src/storage/entry.rs]: `value` ?

**Comment by DarkEld3r** [comment]
Fixed. :upside_down_face:

**Comment by DarkEld3r** [comment]
I have created a separate issue for tests: https://jira.bf.local/browse/ECR-1840

**Pull request merged**